README.md

```diff
@@ -130,52 +130,46 @@ git clone https://github.com/pytti-tools/pytti-notebook
-      git clone https://github.com/pytti-tools/pytti-core
-
-      # Download research code from reference repositories
-      git clone https://github.com/openai/CLIP
-      git clone https://github.com/CompVis/taming-transformers
-
-      # Download research code modified to integrate better with PYTTI
-      git clone https://github.com/pytti-tools/AdaBins
-      git clone https://github.com/pytti-tools/GMA
+      git clone --recurse-submodules https://github.com/pytti-tools/pytti-core
       ```
 
     The end result should be a separate folder for each of pytti and the downloaded models. You should now have a folder structure that looks something like this:
          
             ├── pytti-notebook
-            │   ├── AdaBins
-            │   ├── CLIP
             │   ├── config
-            │   ├── GMA
             │   ├── images_out
             │   ├── pretrained
             │   ├── pytti-core
-            │   ├── taming-transformers
             │   └── videos
 
-
   3. Download the file `AdaBins_nyu.pt` from google drive. 
 
-          # If you would prefer to do this step in the browser, just visit the URL.
+      1. Run the following command to determine the directory where you should put the model
 
-          gdown -O ./pretrained/ https://drive.google.com/uc?id=1lvyZZbC9NLcS8a__YPcUP7rDiIpbRpoF
+          ```
+          python -c "import os; os.path.expanduser('~/.cache/adabins/')"
+          ```
+          on colab, this would evaluate to: `/root/.cache/adabins/`
 
-          # If you get an error saying this file is not available because it's been accessed too many times or whatever, try this alternative URL:
+      2. Download the model to the directory indicated by the output of the previous step. Update the command below to use the correct path.
+          ```
+          # If you would prefer to do this step in the browser, you can download by visiting the URL. You will have to move the file to the correct location afterwards.
 
-          gdown -O ./pretrained/ https://drive.google.com/uc?id=1zgGJrkFkJbRouqMaWArXE4WF_rhj-pxW
+          gdown -O /root/.cache/adabins/ https://drive.google.com/uc?id=1lvyZZbC9NLcS8a__YPcUP7rDiIpbRpoF
 
-      If you downloaded manually, move `AdaBins_nyu.pt` into the `pytti-notebo/pretrained` subdirectory.
+          # If you get an error saying this file is not available because it's been accessed too many times or whatever, try this alternative URL:
+
+          gdown -O /root/.cache/adabins/ https://drive.google.com/uc?id=1zgGJrkFkJbRouqMaWArXE4WF_rhj-pxW
+          ```
 
 4. Install the cloned code into your python environment
 
     ```
     # Install research code
-    pip install ./AdaBins
-    pip install ./CLIP
-    pip install ./GMA
-
-    # Install pytti
+    pip install ./pytti-core/vendor/AdaBins
+    pip install ./pytti-core/vendor/CLIP
+    pip install ./pytti-core/vendor/GMA
+    pip install ./pytti-core/vendor/taming-transformers
     pip install ./pytti-core
     ```
```

Comment:
I'm sure there's a way to get setuptools to take care of this for me, just need to figure it out. Maybe seek support on a python discord or setuptools listerv or whatever